src/main/java/org/jboss/xavier/integrations/jpa/service/FlagAssessmentService.java

```diff
@@ -3,19 +3,21 @@
 import org.jboss.xavier.analytics.pojo.output.workload.summary.FlagAssessmentModel;
 import org.jboss.xavier.integrations.jpa.repository.FlagAssessmentRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Component;
 
-import java.util.List;
-
 @Component
 public class FlagAssessmentService
 {
     @Autowired
     FlagAssessmentRepository flagAssessmentRepository;
 
-    public List<FlagAssessmentModel> findAll()
-    {
-        return flagAssessmentRepository.findAll();
+    public Page<FlagAssessmentModel> findAll(int page, int size) {
```

Comment:
Since the assessments are shown in the Workload Summary Report I think it should be covered in this other PR https://github.com/project-xavier/xavier-integration/pull/114 and not here. Could you please just remove your changes to the '/assessments' endpoints. I'll take your tests and put it in https://github.com/project-xavier/xavier-integration/pull/114

Reply:
I dont understand what you mean